pep-3119.txt

```diff
@@ -115,7 +115,7 @@ documentation for the ABC.  These standard semantic definitions are
 not enforced, but are strongly recommended.
 
 Like all other things in Python, these promises are in the nature of a
-gentlemen's agreement, which in this case means that while the
+friendly agreement, which in this case means that while the
```

Comment:
“mutual understanding” ?

Reply:
Hmm, that could work, albeit with a bit of rewriting. Retaining "agreement" as the second word would be more of a drop-in replacement, though perhaps "understanding" is more what you intend here? As for the first word, perhaps "non-binding" or "informal" better capture the desired meaning? Up to you.

Reply:
I had `informal agreement` in my head too.

Reply:
> I had `informal agreement` in my head too.

That's probably the closest match for the original word, for what its worth